src/vulnpy/trigger/xpath.py

```diff
@@ -0,0 +1,45 @@
+import lxml.etree
+import xml.etree.ElementTree as ET
+from io import StringIO
+
+xml = (
+    "<planets>"
+    "<planet>\
+    <name>Kepler</name>\
+</planet>\
+<planet>\
+    <name>Kepler</name>\
+</planet>\
+<planet>\
+    <name>Newton</name>\
+</planet>"
+    "</planets>"
+)
+
+
+def do_lxml_etree_findall(user_input):
+
+    tree = lxml.etree.parse(StringIO(xml))
+
+    tree.xpath('/planet[name[text()="Kepler"]]')
+    root = tree.getroot()
+    root.findall(user_input)
+
+    return 200
```

Comment:
these aren't really view functions, and are intended to be useful outside of the context of a web application. Instead of returning 200, can you return a result relevant to the trigger function?

For example, you could return a stringified version of the return value from `root.findall`. Same for all functions here